wire-schema/src/main/java/com/squareup/wire/schema/Linker.java

```diff
@@ -27,14 +27,14 @@
 import java.util.Map;
 
 /** Links local field types and option types to the corresponding declarations. */
-final class Linker {
+public final class Linker {
   private final ImmutableList<ProtoFile> protoFiles;
   private final Map<String, Type> protoTypeNames;
   private final Multimap<String, String> imports;
   private final List<String> errors;
   private final List<Object> contextStack;
 
-  Linker(Iterable<ProtoFile> protoFiles) {
+  public Linker(Iterable<ProtoFile> protoFiles) {
```

Comment:
If we're going to expose this type I would move creation into a static factory method just in case.

Reply:
works for me

Reply:
nvm, added a top-level extension function instead to return a linked Schema from a list of ProtoFile, mimicking previous behavior without weakening access.